lambda-notes/src/components/List/NoteList.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import Note from './Note'
+import {NoteListDiv, NoteContainer, NoteTitle} from '../../Styles/NoteViewStyles';
+import '../../Styles/Notes.css';
```

Comment:
In this and many other files You try to import and refer to files that do not exist at all. This must be fixed.